a-a-ron-SLC.md

```diff
@@ -0,0 +1,3 @@
+Favorite restaurants in SLC are:
```

Comment:
@a-a-ron it might be a good idea to include a header with the name of your city!